pkg/cli/secret_encrypt.go

```diff
@@ -33,10 +34,25 @@ func (e *Encrypt) Run(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	if len(args[0]) > 4096 {
+	if len(args) == 1 && len(args[0]) > 4096 {
 		logrus.Fatal("Length of string data is too long to encrypt. Must be less than 4096 bytes.")
 	}
 
+	var q []*survey.Question
+	if len(args) == 0 {
+		q = append(q, &survey.Question{
+			Name:     "plaintext",
+			Prompt:   &survey.Password{Message: "Data to encrypt"},
+			Validate: survey.MaxLength(4096),
+		})
+
+		args = append(args, "")
+	}
+
+	if err := survey.Ask(q, &args[0]); err != nil {
```

Comment:
This is a bit tricky that you always ask even if len(args) == 1.  I guess maybe that works?  It would be more straight forward if you only ask when length == 0 and then you append and the rest of the code assume args length is 1

Reply:
There are no questions to ask? It seems to be the same way it's handled in login.